docs/blog/2021-community-update.rst

```diff
@@ -0,0 +1,121 @@
+.. post:: Aug 13, 2020
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update
+===============================
+
+Hey there documentarians!
+
+The newsletter team is taking August off this year,
+but there's still plenty going on in the community.
+This update covers what else is going on behind the scenes while they take a much deserved break.
+
+We're looking forward to seeing everyone online for our `Prague <http://www.writethedocs.org/conf/prague/2021/>`_ on **October 3-5** in the CEST timezone.
+We're also getting excited about our upcoming virtual `Australia & India <http://www.writethedocs.org/conf/australia/2021/>`_ conference on **December 2-3** in the AEDT timezone.
+The talks were just announced for the Prague conference, and the `Australia & India conference CFP <https://www.writethedocs.org/conf/australia/2021/cfp/>`_ **is still open if you have ideas to present.**
+
+We also had our second virtual Portland conference in April this year.
+It was another success with almost 700 people attending from all over the world.
+The `videos <https://www.writethedocs.org/videos/portland/2021/>`_ from the conference are now available online.
+
+Write the Docs Enhancement Proposals (WEPs)
+-------------------------------------------
+
+We have had a WEPs for a little while now,
+but they haven't seen a ton of traction in the tumultuous last 18 months.
+There are a couple of exciting updates to report though.
+
+We have approved our `Slack Moderation WEP`_, which defines the process for moderating Slack much more explicitly.
+This establishes explicit process in order to grow our moderation team,
+and reduce confusion about the process involved.
+We strive to keep the community friendly and inviting,
+and moderation is a necessary part of that work.
+
+We have also just launched a new `organizational structure and governance WEP`_,
+which will help to shape the structure of how Write the Docs operates.
+This WEP will define all the various teams in the community,
+as well as how decisions are made at the organizational level.
+The proposal adds the concept of a Community Council,
+which will be made up of the team leads of all the community teams.
+The Community Council will make many decisions around the community,
+allowing more people to share in the official leadership structure.
+
+We hope that more people in our community will participate in the WEP process,
+so please read through the proposal and give us feedback.
+
+.. _Slack Moderation WEP: https://github.com/writethedocs/weps/blob/master/accepted/WEP0002.rst#wep2---slack-community-moderation-procedures
+.. _organizational structure and governance WEP: https://github.com/writethedocs/weps/pull/5
+
+Write the Docs store
```

Comment:
Do we want to explicityl say "swag store" or "merchandise" or something to indicate that this is just for swag?

Reply:
Seems reasonable. I don't have a strong feeling on the branding of it -- the current site says "Community Shop" so maybe lets use that? 

```suggestion
Write the Docs community shop
```

Reply:
I like Community Shop more than swag. Merchandise could also work.